tests/v2/validation/rbac/rbac_namespace_move_test.go

```diff
@@ -0,0 +1,200 @@
+package rbac
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type NameSpaceMoveTestSuite struct {
+	suite.Suite
+	client               *rancher.Client
+	standardUser         *management.User
+	standardUserClient   *rancher.Client
+	session              *session.Session
+	cluster              *management.Cluster
+	adminProject         *management.Project
+	additionalUser       *management.User
+	additionalUserClient *rancher.Client
+	standardUserPO       *management.Project
+	standardUserPR       *management.Project
+	steveAdminClient     *v1.Client
+	steveNonAdminClient  *v1.Client
+	namespace            string
+	clusterName          string
+}
+
+func (rb *NameSpaceMoveTestSuite) TearDownSuite() {
+	rb.session.Cleanup()
+}
+
+func (rb *NameSpaceMoveTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	rb.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(rb.T(), err)
+
+	rb.client = client
+
+	//Get cluster name from the config file and append cluster details in rb
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(rb.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(rb.client, clusterName)
+	require.NoError(rb.T(), err, "Error getting cluster ID")
+	rb.cluster, err = rb.client.Management.Cluster.ByID(clusterID)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *NameSpaceMoveTestSuite) CreateProjects() {
+	newUser, err := createUser(rb.client, standardUser)
+	require.NoError(rb.T(), err)
+	rb.standardUser = newUser
+	rb.T().Logf("Created user: %v", rb.standardUser.Username)
+	rb.standardUserClient, err = rb.client.AsUser(newUser)
+	require.NoError(rb.T(), err)
+
+	//Create project as a Project Owner
```

Comment:
Just an inquiry, not necessarily something to fix. In the RBAC test suite, is it common for comments to be relevant instead of logging? In the provisioning test suite, we usually have logging as opposed to comments, so this is more just for my understanding.

Reply:
Although there is currently no standardized approach for RBAC tests, we initially began with comments and following that practice. :)